package.json

```diff
@@ -72,5 +73,6 @@
   "dependencies": {
     "lodash.debounce": "^4.0.8",
     "prop-types": "^15.6.2"
-  }
+  },
+  "peerDependencies": {}
```

Comment:
Seems like this may have been added to, then a dependency removed? Either way, we should nuke it now.

Reply:
It's auto generated with `yarn add` and `yarn remove`. I'm okay with leaving it for the sake of never seeing it again in a future diff.